src/components/result-ui/summary.js

```diff
@@ -1,6 +1,8 @@
 import React, { useContext } from 'react';
 import {LinkElement} from "../link";
 import {FieldFocusContext} from "../../contexts/field-focus";
+import {checkIncludesPath, isEmpty} from "../../utils";
+import {LocationContext} from "../../contexts/location";
```

Comment:
nit: remember to run prettier :)

Reply:
🤦‍♂  Sorry!

Reply:
Wait, I just did and it didn't say anything. What's wrong here?

Reply:
not sure- it's supposed to put spaces around object assignments like 
```suggestion
import { LocationContext } from "../../contexts/location";
```